lib/styles/design-system.scss

```diff
@@ -50,7 +50,9 @@ textarea {
 }
 
 html.thick-scrollbar {
-	@include scrollbar(15px, var(--raw-gray-100));
+	@media #{breakpoint-s()} {
```

Comment:
what about the tablet?

Reply:
right :)
https://bethinkteam.slack.com/archives/C012R8RM3A4/p1682688253072869

Reply:
> Dopytać UX co z tabletem